bin/latex.py

```diff
@@ -144,20 +141,50 @@ def get_environment():
     return env
 
 
+def build_latex_pdf(builddir, tex_path, problem_path=None):
+    env = get_environment()
+    run_once = getattr(config.args, '1', False)
+    watch = getattr(config.args, 'watch', False)
+    ret = util.exec_command(
+        ['latexmk', '-cd', '-g', '-pdf',
+         '-pdflatex=pdflatex -interaction=nonstopmode -halt-on-error']
```

Comment:
This would be easier to parse (as a human) with double quotes around the 'argument' part (after the `=`). Can you add that, if it works at all.

Reply:
I had tried this already, but it doesn't work :upside_down_face: 

In a shell, when you would write `-pdflatex="pdflatex -options..."`, the shell interprets this by appending the string to the rest of the argument, and it would basically be the same as writing `-pdflatex=pdflatex\ -options...`. Thus, when using a shell, the quotes are never passed to the `latexmk` command, since they are interpreted by the shell. Therefore, writing extra quotes on this line in Python will break stuff.